src/components/App.js

```diff
@@ -76,6 +76,22 @@ class App extends React.Component {
     });
   };
 
+  displayAllOfType = (type) => {
+    if (type === "Venues") {
+      this.setState({
+        searchResults: searchBars('', this.state.bars)
```

Comment:
so search[Shows,Bars,Queens] is being imported from our helper file, but I don't see the changes made to the file on this PR. Was it challenging to set up that logic?

Reply:
That was part of Tiffany and my work this morning. We used it in the gallery to search only the shows, and it turned out to be the perfect solution for the "Show All" functionality. We basically made a function that searches each of the three card types and then called them all in the main search function.